module/platform/darwin/HardwareIO/src/Convert.hpp

```diff
@@ -80,30 +80,30 @@ namespace module::platform::darwin {
             return (value & (1 << bit)) == (1 << bit);
         }
 
-        static float accelerometer(const uint16_t value);
-        static float gyroscope(const uint16_t value);
-        static float voltage(const uint8_t value);
-        static float fsrForce(const uint16_t value);
-        static float fsrCentre(const bool left, const uint8_t value);
+        static float accelerometer(uint16_t value);
+        static float gyroscope(uint16_t value);
+        static float voltage(uint8_t value);
+        static float fsrForce(uint16_t value);
+        static float fsrCentre(bool left, uint8_t value);
 
         static std::tuple<uint8_t, uint8_t, uint8_t> colourLED(uint16_t value);
         static uint16_t colourLEDInverse(uint8_t r, uint8_t g, uint8_t b);
 
-        static float gain(const uint8_t value);
-        static uint8_t gainInverse(const float value);
+        static float gain(uint8_t value);
+        static uint8_t gainInverse(float value);
 
-        static float servoPosition(const uint8_t id, const uint16_t value);
-        static uint16_t servoPositionInverse(const uint8_t id, const float value);
+        static float servoPosition(uint8_t id, uint16_t value);
+        static uint16_t servoPositionInverse(uint8_t id, float value);
 
-        static float servoSpeed(const uint8_t id, const uint16_t value);
-        static uint16_t servoSpeedInverse(const float value);
+        static float servoSpeed(uint8_t id, uint16_t value);
+        static uint16_t servoSpeedInverse(float value);
 
-        static float torqueLimit(const uint16_t value);
-        static uint16_t torqueLimitInverse(const float value);
+        static float torqueLimit(uint16_t value);
+        static uint16_t torqueLimitInverse(float value);
 
-        static float servoLoad(const uint8_t id, const uint16_t value);
+        static float servoLoad(uint8_t id, uint16_t value);
 
-        static float temperature(const uint8_t value);
+        static float temperature(uint8_t value);
```

Comment:
Is it more performant to pass by value rather than const reference?

Reply:
I think clang-tidy takes issue with this because the `const` applies here to a value (and therefore is irrelevant to the signature), so having it at all is misleading. I should change these to be `const` references though, to match the rest of the codebase.

Reply:
yes